src/nlohmann/json.hpp

```diff
@@ -0,0 +1,13003 @@
+/*
```

Comment:
Why does this file need to be added to the repo? It's already added to the tarball in `local.mk`.

Reply:
The problem is that currently you can't build Git checkout of Nix on a system without Nix, and the only thing which breaks that is json.hpp because it's added from nixpkgs. With this I can build Nix from scratch in Ubuntu/Fedora Docker container.

Reply:
This change has landed in master 7f2c324ed18cba4004ff89dfd84cf2df979b2571